install.sh

```diff
@@ -11,18 +11,23 @@ name=languageclient
 
 try_curl() {
     command -v curl > /dev/null && \
-        curl --fail --location "$1" --output bin/$name
+        curl --fail --location "$1" --output bin/$name && \
+        curl --fail --location "$1.sha256" --output bin/$name.sha256
```

Comment:
Is it needed to cleanup previous sha256 before downloading? Like the current cleanup of bin file.

Reply:
With the `--output` from `curl` and `--output-document` from `wget` the file will be overwritten so I don't think its useful but if you think its safer let me know